### DISCUSSION TITLE
Encaixar o GRID no Layoult do celular

### DISCUSSION THREAD

**Discussion by jescatolini** [jescatolini/SpaceCream_Stage03_Rocketseat#4] [Q&A] [answered]
Eu criei esse aplicativo para treinar o display grid, porém tive que bular com CSS para encaixar a página perfeitamente com margin;

**Comment by SurfacDev** [comment, marked as answer]
Você não fez da meneira incorreta porque as vezes temos que achar somente a solução do problema, mas você poderia usar o stretch,
